templates/org/team/repositories.tmpl

```diff
@@ -7,7 +7,7 @@
 			{{template "org/team/sidebar" .}}
 			<div class="ui ten wide column">
 				{{template "org/team/navbar" .}}
-				{{$canAddRemove := and $.IsOrganizationOwner (not (eq $.Team.LowerName "owners"))}}
+				{{$canAddRemove := and $.IsOrganizationOwner (not $.Team.IncludesAllRepositories)}}
```

Comment:
Why this change?

Reply:
If all repositories shall be included, the page shall be adopted so repositories can't be removed or added. About owners, see my comment above.